providers/dns/oraclecloud/README.md

```diff
@@ -12,17 +10,12 @@ export OCI_REGION="us-phoenix-1"
 export OCI_COMPARTMENT_OCID="ocid1.tenancy.oc1..secret"
 ```
 
-in Fish
-
 ```
-set IFS
-set -x OCI_PRIVKEY_BASE64 (base64 ~/.oci/oci_api_key.pem)
-set IFS \n" "\t
-
-set -x OCI_PRIVKEY_PASS 'secret'
-set -x OCI_TENANCY_OCID 'ocid1.tenancy.oc1..secret'
-set -x OCI_USER_OCID 'ocid1.user.oc1..secret'
-set -x OCI_PUBKEY_FINGERPRINT '00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00'
-set -x OCI_REGION 'us-phoenix-1'
-set -x OCI_COMPARTMENT_OCID 'ocid1.tenancy.oc1..secret'
+export OCI_PRIVKEY=`base64 ~/.oci/oci_api_key.pem`
```

Comment:
The previous documentation supported `fish`, but the new documentation here does not indicate why or how one would use the other configuration. Possibly add some comments to the README on how it works?

Reply:
It's not required by the new configuration way (no base64)

Reply:
ooops forget :smile_cat: